_config.yml

```diff
@@ -101,6 +101,28 @@ defaults:
       toc_min: 2
       toc_max: 4
 
+  # Exlcude from sitemap
+  - scope:
+      path: "assets/**"
+    values:
+      sitemap: false
+  - scope:
+      path: "fonts/**"
+    values:
+      sitemap: false
```

Comment:
we can remove this folder in a follow-up if we are ok with https://github.com/docker/docker.github.io/pull/15246 so all resources will be in `assets`.

Reply:
https://github.com/docker/docker.github.io/pull/15246 merged so remove this folder fro mexclusion